packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -66,8 +76,8 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
     })
 
     it('can translate, create, and publish a draft of a policy intent', async () => {
-      const start_date = now() + 50000
-      const end_date = now() + 500000
+      const start_date = now() + hours(1)
```

Comment:
Might add a comment here describing why it's being offset by 1 hour (to avoid the issue with start_date needing to be 20 minutes from now I assume).